File: CS/Q205284/MyPopupControlContainer.cs
using DevExpress.XtraBars;
using System.ComponentModel;
using DevExpress.XtraBars.Controls;
using DevExpress.LookAndFeel;
using System;
using System.Reflection;

namespace DXSample {
    public class MyPopupControlContainer : PopupControlContainer {
        public MyPopupControlContainer() : base() { }
        public MyPopupControlContainer(IContainer container) : base(container) { }
        private UserLookAndFeel temp;
        protected override PopupContainerBarControl CreateSubControl(BarManager manager) {
            if (temp == null) temp = new UserLookAndFeel(this);
            temp.Assign(LookAndFeel);
            return base.CreateSubControl(manager);
        }
        protected override PopupContainerBarControl CreatePopupContainerBarControl(BarManager manager) {
            LookAndFeel.Assign(temp);
            return base.CreatePopupContainerBarControl(manager);
        }
    }
}

File: VB/Q205284/MyPopupControlContainer.vb
Imports Microsoft.VisualBasic
Imports DevExpress.XtraBars
Imports System.ComponentModel
Imports DevExpress.XtraBars.Controls
Imports DevExpress.LookAndFeel
Imports System
Imports System.Reflection

Namespace DXSample
	Public Class MyPopupControlContainer
		Inherits PopupControlContainer
		Public Sub New()
			MyBase.New()
		End Sub
		Public Sub New(ByVal container As IContainer)
			MyBase.New(container)
		End Sub
		Private temp As UserLookAndFeel
		Protected Overrides Function CreateSubControl(ByVal manager As BarManager) As PopupContainerBarControl
			If temp Is Nothing Then
				temp = New UserLookAndFeel(Me)
			End If
			temp.Assign(LookAndFeel)
			Return MyBase.CreateSubControl(manager)
		End Function
		Protected Overrides Function CreatePopupContainerBarControl(ByVal manager As BarManager) As PopupContainerBarControl
			LookAndFeel.Assign(temp)
			Return MyBase.CreatePopupContainerBarControl(manager)
		End Function
	End Class
End Namespace